src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,57 @@
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $clock-height;
+  width: $clock-width;
+  position: relative;
+
+  border: 5px solid $clock-color;
+  border-radius: 50%;
+  box-shadow: 0 -15px 15px rgba(255, 255, 255, 0.05),
+    inset 0 -15px 15px rgba(255, 255, 255, 0.01),
+    0 15px 15px rgba(0, 0, 0, 0.3),
+    inset 0 15px 15px rgba(0, 0, 0, 0.3);
+
+  &__center {
+    position: absolute;
+    height: $center-height;
+    width: $center-width;
+    background-color: $center-color;
+    border-radius: 50%;
+  }
+
+  &__minute {
+    position: absolute;
+    height: $minute-hand-height;
+    width: $minute-hand-width;
+    background-color: $minute-hand-color;
+    border-radius: 6px 6px 0 0;
+    bottom: 50%;
+    transform-origin: bottom;
+    /* transform: translateY($minute-hand-height / -2); */
+    animation: move 3600s linear infinite;
+  }
+
+  &__seconds {
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $seconds-hand-width;
+    background-color: $seconds-hand-color;
+    border-radius: 6px 6px 0 0;
+    bottom: 50%;
+    transform-origin: bottom;
+    /* transform: translateY($seconds-hand-height / -2); */
+    animation: move 60s linear infinite;
+  }
```

Comment:
&__minute and &__seconds have almost same styles.
make mixin and pass to it necessary parameters.